components/Form/AddFlightForm.js

```diff
@@ -0,0 +1,322 @@
+import {
+  StyledForm,
+  Card,
+  StyledAddButton,
+  StyledHeading,
+  InputLabel,
+  InputBox,
+  InputField,
+  StyledInfoText,
+} from "./AddFlightForm.styled";
+import { useState } from "react";
+import useLocalStorageState from "use-local-storage-state";
+import { flights } from "@/data/septemberFlights";
+import { useRouter } from "next/router";
+
+export default function AddFlightForm() {
+  const [flightsOfInterest, setFlightsOfInterest] = useLocalStorageState(
+    "flightsInfo",
+    { defaultValue: flights }
+  );
+
+  const router = useRouter();
+
+  const [isFocused, setIsFocused] = useState(false);
+
+  const handleFocus = () => setIsFocused(true);
+  const handleBlur = () => setIsFocused(false);
+
+  function handleSubmit(event) {
+    event.preventDefault();
+    const formData = new FormData(event.target);
+    const flightData = Object.fromEntries(formData);
+
+    setFlightsOfInterest([...flightsOfInterest, flightData]);
+    router.push("/flight_list");
+    event.target.reset();
+    event.target.flight_iata.focus();
```

Comment:
Come to think of it, you can drop these lines since you navigate away from the form page.
```suggestion
```